src/JobAwareTrait.php

```diff
@@ -96,4 +100,20 @@ public function execute($name, $workload, $background = true, $priority = Gearma
         return $response;
     }
 
+    /**
+     * Returns an array with all configured Gearman servers.
+     *
+     * @throws \Cake\Core\Exception\Exception
+     * @return array Flat array with servers as found in Configure
+     */
+    protected static function getServers()
```

Comment:
Not yet even though I did phpcs against "our" sniffer. Next up will be the travis and all fixes.